source/Octopus.Tentacle.CommonTestUtils/Builders/LatestStartScriptCommandBuilder.cs

```diff
@@ -1,6 +1,13 @@
-﻿namespace Octopus.Tentacle.CommonTestUtils.Builders
+﻿using Octopus.Tentacle.Contracts;
+using Octopus.Tentacle.Contracts.Builders;
+
+namespace Octopus.Tentacle.CommonTestUtils.Builders
 {
     public class LatestStartScriptCommandBuilder : StartScriptCommandV3AlphaBuilder
     {
+        public LatestStartScriptCommandBuilder()
+        {
+            WithIsolation(ScriptIsolationLevel.NoIsolation);
```

Comment:
This ensures there is no change to the StartScriptComamnd being used by the Tests